.github/workflows/build.yml

```diff
@@ -240,3 +240,19 @@ jobs:
         ./tools/travis/run-luacheck-${{ matrix.os }}.sh
       shell: bash
 
+
+
+  doc_check:
+
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-16.04 
+      
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: false
+    - name: all_docks_linked
+      run: ./tools/check_docs.sh
```

Comment:
If the idea is to add future checks to the _same_ file then I'm ok with `check_docs.sh`. If not, then it would have to be more specific.

Reply:
Let's keep this in mind and adapt as required if more checks are added.
I could imagine a linkcheck of some kind.
I also would like to extract Lua samples and at least syntax check them, but that would go in the Lua compile step.